docs/source/getting-started/running-notebooks/index.rst

```diff
@@ -0,0 +1,139 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With a Databricks managed instance of MLflow, you have two options on how you'd like to proceed with using the managed Tracking Server instance. You can 
+import any notebooks that you'd like and use them from within CE directly, or you can execute the tutorials locally from your computer and simply login to CE 
+from within your notebook. 
+
+**Importing Notebooks directly into CE**
+
+Once you're at the main page of CE, you can import any of the notebooks within this tutorial by navigating to the "Workspace" tab on the left. Click that link to 
+expand the file navigation pane. 
+From there, navigate to Users/<you> and you can right click to bring up the "Import" option. 
+
+Here is what the import dialog should look like if you're going to directly import a notebook from the MLflow documentation website:
+
+.. figure:: ../../_static/images/tutorials/introductory/import-notebook-into-ce.png
+    :alt: Databricks CE import Notebook from MLflow docs website
+    :width: 100%
+    :align: center
+
+    Importing a tutorial directly from the MLflow docs website
+
+At this point, you can simply just run the tutorial. Any calls to MLflow for creating experiments, initiating runs, logging metadata, and saving artifacts will 
+be fully managed for you. Your logging history will appear within the MLflow UI.
+
+**Running locally and using CE as a remote tracking server**
+
+In order to stay within the comfortable confines of your local machine and still have the use of the managed MLflow Tracking Server, you can simply add a 
+call to ``mlflow.login()`` to your notebook. This will generate a prompt for you to enter your **username** and **password** that were defined when creating your 
+account on CE. Note: your username will be the **email address** that you used when signing up. 
+
+.. code-block:: python
+
+    import mlflow
+
+    mlflow.login()
+
+After the login process is done, you will simply have to set your MLflow Tracking URI to the instance that you just logged in to. It's made fairly easy for you:
+
+.. code-block:: python
+
+    mlflow.set_tracking_uri("databricks")
+
+
+At this point, you're ready to go! You can run any of the tutorials locally and they will log to the managed MLflow Tracking Server.
+
+Self-managed MLflow
+-------------------
+
+If you have your own managed MLflow server that you'd like to use, you can simply set the MLflow Tracking URI to point to your server.
+If there are additional configurations or access restrictions for connecting, please contact the group that manages your MLflow deployment for more information.
+
+To set the MLflow Tracking URI directly (assuming that you don't have additional security validation set up), all you need to do is, from within a notebook:
+
+.. code-block:: python
+
+    mlflow.set_tracking_uri(
+        "http://<your-mlflow-server>:<the port number that is configured to accept traffic>"
+    )
+
+
+Local Tracking Server
```

Comment:
This seems like the easiest way to run notebooks.

Reply:
I agree, but the directive from PM is to point people to CE first and foremost

Reply:
got it